### PR TITLE
fix(flow): remove repeat option from children

### PR DESCRIPTION
**Pull request by roggervalf**
fixes https://github.com/taskforcesh/bullmq/issues/851

### CODE REVIEW

File: src/interfaces/flow-job.ts [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+  opts?: Omit<JobsOptions, 'parent' | 'repeat'>;
+  children?: FlowChildJob[];
+}
+
+export interface FlowJob extends BaseFlowNode {
   name: string;
   queueName: string;
   data?: any;
   prefix?: string;
   opts?: Omit<JobsOptions, 'parent'>;
```
Comment (manast) [inline src/interfaces/flow-job.ts:21]: hmm, here we still allow repeat, is it correct?

Comment (roggervalf) [reply src/interfaces/flow-job.ts:21]: I was understanding from this comment https://github.com/taskforcesh/bullmq/issues/851#issuecomment-957025297 that parent could have repeat option, but children not :thinking:

Comment (manast) [reply src/interfaces/flow-job.ts:21]: No, what I meant is that as soon as you have parent/child properties in options we should not allow also having repeat options, as they need to be mutually exclusive, either you use repeat or you use flows but not both.

Comment (roggervalf) [reply src/interfaces/flow-job.ts:21]: Got it, I'll update this fix

**Review by manast — APPROVED** [review]

**Pull request merged**